tests/NuGet.Services.AzureSearch.FunctionalTests/Relevancy/V3RelevancyFunctionalTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace NuGet.Services.AzureSearch.FunctionalTests
+{
+    public class V3RelevancyFunctionalTests : NuGetSearchFunctionalTestBase
+    {
+        public V3RelevancyFunctionalTests(CommonFixture fixture)
+            : base(fixture)
+        {
+        }
+
+        [Fact]
+        public async Task Json()
+        {
+            var results = await SearchAsync("json");
+
+            Assert.Equal("json", results.First());
+            Assert.Equal("newtonsoft.json", results.Skip(1).First());
```

Comment:
nit: elementas is clearer IMO

or just return something that is index-able

Reply:
Agreed, this will get painful fast. The results are a read-only list, I'll use indexing

Reply:
Updated to use indexing